service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storage/CreateAzureStorageStepTest.java

```diff
@@ -1,5 +1,10 @@
 package bio.terra.workspace.service.resource.controlled.cloud.azure.storage;
 
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
@jaycarlton This import ordering doesn't seem to be enforced by `google-java-format`, as when I reformat wit `google-java-format` in IntelliJ I don't get these changes. Therefore it seems that I need to run spotless locally also. What is the command to do that, and do you know if there is a way to set up IntelliJ's formatting to automatically use spotless?

Reply:
To run spotless, do `./gradlew spotlessApply` after this is merged.

It's possible to set up IntelliJ so that it runs `spotlessApply` every time a file is saved.  @melissachang is the only one I know who's done that.

Also, import ordering seems to work for me when I let IJ handle all the imports. I.e. I never manually add an import statement, and they seem to go to the right place FWIW.

Reply:
Have Intellij run spotless on file save:

https://github.com/diffplug/spotless/issues/200#issuecomment-1063519752